ldi-core/sparql-construct/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/SparqlConstructTransformer.java

```diff
@@ -8,17 +8,17 @@
 
 public class SparqlConstructTransformer implements LdiTransformer {
 	private final Query query;
-	private final boolean inferMode;
```

Comment:
I am not sure that we want this as inference is an actual concept within rdf:
https://jena.apache.org/documentation/inference/#overview

Reply:
Isn't this a name you originally implemented? 😅